lnrpc/neutrinorpc/neutrino_server.go

```diff
@@ -192,12 +196,12 @@ func (s *Server) Status(ctx context.Context,
 	in *StatusRequest) (*StatusResponse, error) {
 
 	if s.cfg.NeutrinoCS == nil {
-		return &StatusResponse{}, nil
+		return nil, nil
```

Comment:
This will result in an encoding error if both are `nil`. So this should return an error instead (which makes sense if there is no Neutrino chain source).

Reply:
I should have noticed that myself. Fixed.

Reply:
I will add itests for all these functions after  this is merged.